copper-server/src/main/java/com/pogeyan/cmis/actors/DiscoveryActor.java

```diff
@@ -72,9 +79,9 @@ private JSONObject getContentChanges(QueryGetRequest request) throws CmisRuntime
 		LOG.info(
 				"Method name: {}, get latest content changes using this id: {}, repositoryId: {}, includeAcl: {}, includePolicyIds: {}",
 				"getContentChanges", changeLogTokenHolder, request.getRepositoryId(), includeAcl, includePolicyIds);
-		ObjectList changes = CmisDiscoveryService.Impl.getContentChanges(request.getRepositoryId(),
-				changeLogTokenHolder, includeProperties, filter, orderBy, includePolicyIds, includeAcl, maxItems, null,
-				request.getUserObject());
+		ObjectList changes = CmisDiscoveryService.Impl.getContentChanges((String) baggage.get(TRACINGID),
```

Comment:
baggage.get(TRACINGID) might be null as well if PerfMode is set to 0, Please make proper checks or higher order functions here so we can re-use.